DROD/HoldSelectScreen.cpp

```diff
@@ -958,9 +958,18 @@ void CHoldSelectScreen::OnBetweenEvents()
 		// Done downloading the list.
 		CScreen::HideStatusMessage();
 		this->bDownloadList = false;
-		PopulateHoldListBox();
 
 		this->pSelCNetHold = NULL;
+		PopulateHoldListBox();
+
+		UINT caravelNetSelectHoldId = (UINT)g_pTheNet->GetDownloadHold();
```

Comment:
K, thanks, this makes sense.

Can remove the UINT cast here now,

Reply:
Done, missed that comment earlier.